code/game/objects/items/weapons/mop.dm

```diff
@@ -1,3 +1,5 @@
+#define is_cleanable(A) (istype(A, /obj/effect/decal/cleanable) || istype(A, /obj/effect/rune))

```

Comment:
This being a `#define` is iffy; You also left out /obj/effect/overlays from the list.